pkg/util/mount/nsenter_mount.go

```diff
@@ -39,14 +40,31 @@ import (
 //     performed in the host's mount namespace do not propagate out to the
 //     bind-mount in this docker version.
 // 2.  The host's root filesystem must be available at /rootfs
-// 3.  The nsenter binary must be at /nsenter in the container's filesystem.
+// 3.  The nsenter binary must be on the PATH in the container's filesystem.
 // 4.  The Kubelet process must have CAP_SYS_ADMIN (required by nsenter); at
 //     the present, this effectively means that the kubelet is running in a
 //     privileged container.
+// 5.  The volume path used by the Kubelet must be the same inside and outside
+//     the container and be writable by the container (to initialize volume)
+//     contents. TODO: remove this requirement.
 //
 // For more information about mount propagation modes, see:
 //   https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt
-type NsenterMounter struct{}
+type NsenterMounter struct {
+	// the path on the parent filesystem that contains mount, umount, and findmnt
+	hostBinPath string
+}
+
+func NewNsenterMounter() *NsenterMounter {
+	// search for the mount command in other locations besides /usr/bin
```

Comment:
Can you add a comment with justification for expecting all of the binaries to be located in the same directory? (i.e., why not store three path variables, one for each?)